src/group_theory/perm/cycle_type.lean

```diff
@@ -223,6 +224,22 @@ begin
   exact one_lt_two,
 end
 
+lemma subgroup.eq_top_of_mem_swap [decidable_eq α] {H : subgroup (perm α)} {τ : perm α}
```

Comment:
This results in the lemma name `equiv.perm.subgroup.eq_top_of_mem_swap` which isn't going to do anything useful for dot notation - so you may as well remove the `.` to avoid this falsely looking like it is in the subgroup namespace
```suggestion
lemma subgroup_eq_top_of_mem_swap [decidable_eq α] {H : subgroup (perm α)} {τ : perm α}
```